lib/rubocop/cop/style/mutable_constant.rb

```diff
@@ -83,6 +84,7 @@ def on_assignment(value)
         end
 
         def strict_check(value)
+          return if shareable_constant_value?
           return if immutable_literal?(value)
```

Comment:
I find it surprising that the tests started failing when i changed `immutable_literal?(value)` to `!mutable_literal?(value)`.